requirements_test.txt

```diff
@@ -1,6 +1,6 @@
 aioconsole==0.5.1
 black==22.12.0
-flake8=6.0.0
+flake8==6.0.0
```

Comment:
Today in main I replaced these lint/format tools with ruff (same as HA uses), and added a pre-commit hook for it.